README.md

```diff
@@ -2763,7 +2765,7 @@ Other Style Guides
 
 
       getAge(today, birthday) {
-        // ..
+        // ...
```

Comment:
i don't think it's worth the diff churn to add a third period :-) if you wanted to change them to a proper ellipsis, i guess that'd be fine, otherwise revert.

Reply:
this document is so beautiful that I want it to look perfect :)

Reply:
then I decided to fix this inside the same commit while fixing something else.

Reply:
I changed this under commit named Add console.log.

Reply:
Sorry if my original comment wasn't clear - these should be reverted to two dots.

Reply:
Ok, I've recommitted.